src/options.ts

```diff
@@ -1,7 +1,4 @@
-export enum JitterTypes {
-  None = "none",
-  Full = "full"
-}
+export type JitterTypes = "none" | "full";
```

Comment:
Could we adjust the name to use the singular please: `JitterType`

Reply:
done!

Reply:
Thanks